include/mbedtls/pkcs7.h

```diff
@@ -160,21 +155,21 @@ typedef struct mbedtls_pkcs7 {
 mbedtls_pkcs7;
 
 /**
- * \brief          Initialize pkcs7 structure.
+ * \brief          Initialize mbedtls_pkcs7 structure.
```

Comment:
Note: line 45 of pkcs7.c says "Initializes the pkcs7 structure.`

Reply:
I'm mostly concerned about the wording in the public-facing headers - inside `pkcs7.c` it's not so much an issue. I'll fix as I'm updating `mbedtls_config.h` though.